app/Policies/UserPolicy.php

```
@@ -0,0 +1,48 @@
+namespace App\Policies;
+
+use App\Models\User;
+
+class UserPolicy
+{
+    /**
+     * Determine whether the user can view any models.
+     */
+    public function viewAny(User $user): bool
+    {
+        return $user->role->isSuperAdmin();
```

PHPStan (2.1.15)

Cannot call method isSuperAdmin() on int.


(method.nonObject)